Chrome crashing because cannot load libchrome.so

Any clue where I could look to figure out why libchrome.so isn't loaded?

I'm doing a build based off of CM 12 ( Android 5.0.2 ).

```
I/cr_LibraryLoader(10337): Using linker: org.chromium.base.library_loader.LegacyLinker
I/cr_LibraryLoader(10337): Loading chrome from within /system/app/Chrome/Chrome.apk
E/cr_ChromiumAndroidLinker(10337): Open: Could not open libchrome.so in zip file /system/app/Chrome/Chrome.apk: (null)
E/cr_LibraryLoader(10337): Unable to load library: libchrome.so, in: /system/app/Chrome/Chrome.apk
E/NativeInitializationController(10337): Unable to load native library.
E/NativeInitializationController(10337): org.chromium.base.library_loader.ProcessInitException
E/NativeInitializationController(10337):        at org.chromium.base.library_loader.LibraryLoader.loadAlreadyLocked(LibraryLoader.java:271)
E/NativeInitializationController(10337):        at org.chromium.base.library_loader.LibraryLoader.ensureInitialized(LibraryLoader.java:123)
E/NativeInitializationController(10337):        at org.chromium.chrome.browser.init.NativeInitializationController$1.run(NativeInitializationController.java:84)
E/NativeInitializationController(10337): Caused by: java.lang.UnsatisfiedLinkError: Unable to load library: libchrome.so, in: /system/app/Chrome/Chrome.apk
E/NativeInitializationController(10337):        at org.chromium.base.library_loader.LegacyLinker.loadLibraryImpl(LegacyLinker.java:500)
E/NativeInitializationController(10337):        at org.chromium.base.library_loader.Linker.loadLibrary(Linker.java:569)
E/NativeInitializationController(10337):        at org.chromium.base.library_loader.LibraryLoader.loadLibrary(LibraryLoader.java:205)
E/NativeInitializationController(10337):        at org.chromium.base.library_loader.LibraryLoader.loadAlreadyLocked(LibraryLoader.java:250)
E/NativeInitializationController(10337):        ... 2 more
```

--- a/README.md
+++ b/README.md
@@ -136,3 +136,11 @@ Pull requests to add package overrides for more modules is welcome. See `modules
 ### This has currently only been tested on a 5.1 ARM based device
 Your milage may vary.
 
+### Chrome on Lollipop requires an extra patch
+Run these commands:
+```
+cd build
+curl https://raw.githubusercontent.com/opengapps/aosp_build/master/patches/Lollipop/0001-Fix-Chrome.patch | git am -
+```
+
+(Patch only tested on 5.1.1 r37).

--- a/patches/Lollipop/0001-Fix-Chrome.patch
+++ b/patches/Lollipop/0001-Fix-Chrome.patch
@@ -1,0 +1,29 @@
+From ec855ea4e180102a11ee5ecd3345c92477f575fa Mon Sep 17 00:00:00 2001
+From: Arne-Christian Blystad <arne.christian@blystad.me>
+Date: Mon, 11 Apr 2016 11:08:20 +0200
+Subject: [PATCH] Don't extract crazy native libraries. Fixes Chrome.
+
+Chrome uses its own module loader (libcrazy) which it expects to find inside the
+APK, not outside.
+
+Fixes #21
+---
+ core/prebuilt_internal.mk | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/core/prebuilt_internal.mk b/core/prebuilt_internal.mk
+index bc6088f..03bcf1f 100644
+--- a/core/prebuilt_internal.mk
++++ b/core/prebuilt_internal.mk
+@@ -183,7 +183,7 @@ include $(BUILD_SYSTEM)/dex_preopt_odex_install.mk
+ $(built_module) : $(my_prebuilt_src_file) | $(ACP) $(ZIPALIGN) $(SIGNAPK_JAR)
+ 	$(transform-prebuilt-to-target)
+ ifdef extracted_jni_libs
+-	$(hide) zip -d $@ 'lib/*.so'  # strip embedded JNI libraries.
++	$(hide) unzip -Z -1 $@ | grep "lib/" | grep -v "/crazy." | xargs zip -d $@ # strip embedded JNI libraries.
+ endif
+ ifneq ($(LOCAL_CERTIFICATE),PRESIGNED)
+ 	$(sign-package)
+-- 
+2.7.4
+

--- a/patches/Lollipop/README.md
+++ b/patches/Lollipop/README.md
@@ -1,0 +1,7 @@
+# Lollipop patches
+This folder includes patches that should be applied to AOSP in order to ensure 
+everything works correctly/as expected.
+
+## 0001-Fix-Chrome.patch
+
+Applies to `build`. Fixes issues with Chrome.

--- a/patches/README.md
+++ b/patches/README.md
@@ -1,0 +1,1 @@
+# AOSP Patches

